js/src/util/index.js

```diff
@@ -135,20 +135,26 @@ const typeCheckConfig = (componentName, config, configTypes) => {
 }
 
 const isVisible = element => {
-  if (!element) {
+  const eligibleElement = el => el && el.style
+
+  if (!eligibleElement(element)) {
```

Comment:
Should we combine the two `if`s? Maybe add a line break to make it clearer, though.

Reply:
Do you think this looks more clear?

```javascript
if (
  !eligibleElement(element) ||
  (element.tagName !== 'AREA' && element.getClientRects().length === 0)
) {
  return false
}
```

Reply:
It's subjective, I'd say, but to me it's the same plus it's shorter :)